guides/using_blocks_like_functions.md

```diff
@@ -0,0 +1,72 @@
+<script type="module" src="https://gradio.s3-us-west-2.amazonaws.com/3.0.24/gradio.js"></script>
```

Comment:
I debated whether or not to include this in an already existing guide or add it to its own guide.

The two other guides where this could go are:
1. Introduction to Blocks
2. Using Hugging Face Integrations

I don't think this is an introductory concept, so it'd be out of place in the introduction to blocks guide.

It could go in the integrations guide but there's a lot already happening there so I'm afraid this content would get lost.

Ultimately I felt this was a concept worth going into depth so I added it to its own guide.